### PR TITLE
Bump pr/felipe-cabrera/#5193: VS Code productividade infinita

### PR DESCRIPTION
**Pull request by davorpa**
## What does this PR do?
Add resource(s) | Add info | Improve repo

### Description

Pull request at #5193 made by @felipe-cabrera has a stale state with conficts. Conflicts were introduced with a repo reorganization (book files was moved to books folder `free-courses-pt_BR.md`~>`courses/free-courses-pt_BR.md`) made on Dec 2020 (#5199).

I´ll try to solve it here and then go ahead with it.

### Why is this valuable (or not)?
Resolves #5193, Closes #5193, Locks #5193

Also moves Vim to Agnostic section and then reorganize all under `IDE / Editors`, and completes some info on those moved resources.

### How do we know it's really free?
Follow original PR, Felipe is who creates the book.

### For book lists, is it a book? For course lists, is it a course? etc.
It's a book / long tutorial

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates. #5193
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Followup

- Check the output of Travis-CI for linter errors!

### CODE REVIEW

File: books/free-programming-books-pt_BR.md [hunk 1]
```diff
@@ -45,11 +46,25 @@
...
+#### IDE / Editores
+
+* [Visual Studio Code: Produtividade infinita](https://github.com/bylearn/VS-Code-Produtividade-Infinita) - Felipe Cabrera Ribeiro dos Santos
+
+
+#### Vim
+
+* [Editor VI - Guia de Referência](https://aurelio.net/curso/material/vim-ref.html) - Aurelio Marinho Jargas
+* [O Editor de Texto Vim](https://code.google.com/p/vimbook) - Sérgio Luiz Araújo Silva et al.
+* [Vim Avançado](https://aurelio.net/vim/vim-avancado.txt) - Aurelio Marinho Jargas
```
Comment (eshellman) [inline books/free-programming-books-pt_BR.md:62]: this may be a cheatsheet?

Comment (eshellman) [reply books/free-programming-books-pt_BR.md:62]: also a cheatsheet

**Review by eshellman — COMMENTED** [review]
aurelio.net resources are not books

File: books/free-programming-books-pt_BR.md (hunk 1 above)
Comment (davorpa) [reply books/free-programming-books-pt_BR.md:62]: Ok, I'll move it 😉

It was an improving changing http to https

Comment (eshellman) [reply books/free-programming-books-pt_BR.md:62]: Hmmm. No problem putting them on the cheatsheet list; IIRC there are already some non-english entries there.

Comment (davorpa) [reply books/free-programming-books-pt_BR.md]: Done. Check it now again

**Pull request merged**